src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -422,15 +422,19 @@ async def ReadAttribute(self, nodeid: int, attributes: typing.List[typing.Union[
 
         nodeId: Target's Node ID
         attributes: A list of tuples of varying types depending on the type of read being requested:
-            ():                                         Endpoint = *,           Cluster = *,          Attribute = *
-            (Clusters.ClusterA):                        Endpoint = *,           Cluster = specific,   Attribute = *
-            (Clusters.ClusterA.AttributeA):             Endpoint = *,           Cluster = specific,   Attribute = specific
-            (endpoint, Clusters.ClusterA):              Endpoint = specific,    Cluster = specific,   Attribute = *
             (endpoint, Clusters.ClusterA.AttributeA):   Endpoint = specific,    Cluster = specific,   Attribute = specific
+            (endpoint, Clusters.ClusterA):              Endpoint = specific,    Cluster = specific,   Attribute = *
+            (Clusters.ClusterA.AttributeA):             Endpoint = *,           Cluster = specific,   Attribute = specific
+            endpoint:                                   Endpoint = specific,    Cluster = *,          Attribute = *
+            Clusters.ClusterA:                          Endpoint = *,           Cluster = specific,   Attribute = *
+            None:                                       Endpoint = *,           Cluster = *,          Attribute = *
```

Comment:
I would use `('*')` for a convenient represent for a path with all wildcard components here with better readability than `None`, can you add a note here?